### ISSUE TITLE
Build: Previously entered Notes appears only when the user opens it the first time or intermittently

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#81]
URL: https://hcdev.openplus.ca/en/node/166/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Chrome, Firefox
Language: Both

Issue Description: When the user opens a Build for the first time, the Notes entered at the bottom appear only at the first view or intermittently. If the user goes to another page/section and comes back, these are not there anymore. These are comments from the user or other users (under Notes)

Steps to reproduce: On Dashboard, open a Recall/Alert by clicking on Build. There is a Notes section at the bottom of the page. Enter Comment in there and click Save. A few comments can be entered. Go to any other page and then open the same Recall/Alert the same way from the Dashboard. These notes seem to appear only the first time or intermittently.

Expected results: These Notes seem to be important and should be visible every time the user goes to them. Some users might not know that there are Notes in there.

![image](https://user-images.githubusercontent.com/14280062/129248393-56679bba-77df-498a-9c03-a17246618f45.png)

**Comment by qanitah** [comment]
The notes are always visible now.
